### ISSUE TITLE
Question about training loss.

### ISSUE THREAD

**Issue by Guanys-dar** [showlab/Tune-A-Video#86]
Thank you for your excellent work, which has been very inspiring to me. 

I have some questions about the loss function used for fine-tuning your network in the context of your paper. In the paper, you mentioned using 'the **same training objective in standard LDMs**' during fine-tuning. However, in Figure 4 of the paper, it is stated that the network uses a **pixel-wise reconstruction loss**, which seems to compute based on the input video and the reconstructed video instead of the predicted noise. Could you please clarify if I am misunderstanding something?

**Comment by henbucuoshanghai** [comment]
输入输出视频一样？视频拆成一帧帧图片，和同样的句子描述，输入sd，得到原始输入图片？
好像是这样。。

**Comment by henbucuoshanghai** [comment]
他的finetune网络估计就是这样训练的

**Comment by DI-LEE** [comment]
I have a same question!!!!!
please help!
